.github/workflows/pr-ci.yml

```diff
@@ -73,13 +82,25 @@ jobs:
         registry: ghcr.io/${{ github.repository_owner }}
         username: ${{ github.event.pull_request.user.login }}
         password: ${{ secrets.GHCR_PR_TOKEN }}
-      if: github.repository_owner == 'cryostatio'
+    - name: store image amd64
+      id: amd64_image
+      outputs:
+        amd64_image: ${{ steps.push-to-ghcr.outputs.registry-path }}
+      if: matrix.arch == 'amd64'
+    - name: store image arm64
+      id: arm64_image
+      outputs:
+        amd64_image: ${{ steps.push-to-ghcr.outputs.registry-path }}
+      if: matrix.arch == 'arm64'
     - name: Comment test image link
       uses: thollander/actions-comment-pull-request@v1
       with:
         message: |-
-          Test image available:
+          arm64 test image available:
+          ```
+          $ CRYOSTAT_IMAGE=${{ steps.amd64_image.outputs.amd64_image }} sh smoketest.sh
           ```
-          $ CRYOSTAT_IMAGE=${{ steps.push-to-ghcr.outputs.registry-path }} sh smoketest.sh
+          amd64 test image available:
           ```
-      if: github.repository_owner == 'cryostatio'
+          $ CRYOSTAT_IMAGE=${{ steps.arm64_image.outputs.arm64_image }} sh smoketest.sh
+          ```
```

Comment:
Files need new lines: see here for why: https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b

You can enable this automatically when you save by enabling this setting for VSCode:
![image](https://github.com/cryostatio/cryostat/assets/69827283/a6ff2e13-d39f-4618-8869-afe99e61d859)